elyra/metadata/schemas/kfp.json

```
@@ -33,6 +33,19 @@
           "type": "string",
           "format": "uri"
         },
+        "api_endpoint_username": {
+          "title": "Kubeflow Pipelines API Endpoint Username",
+          "description": "The Kubeflow Pipelines API Endpoint username",
```

We should probably lowercase endpoint in the description (same goes for the password property)
```suggestion
          "description": "The Kubeflow Pipelines API endpoint username",
```